kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -220,6 +221,7 @@
     &:focus
       outline: none
       border-color: transparent
+      border-bottom: 2px solid $core-action-normal
```

Comment:
Yeah, agreed that the default doesn't look too hot - looks like the border is implemented using a wrapper div. I was trying (and failed) to more similar to the the existing 'search' box in the Learn app.

The fields in modals (login, edit, add) don't have borders, so adding a 2px bottom border inside the existing border looked really odd to me...